cf-custom-resources/lib/custom-domain.js

```diff
@@ -172,7 +172,7 @@ const writeARecord = async function (
  */
 exports.handler = async function (event, context) {
   var responseData = {};
-  var physicalResourceId;
+  const physicalResourceId = `${event.LogicalResourceId}`;
```

Comment:
Could this be just `event.LogicalResourceId` without wrapped within "`${}`"?

Reply:
yes sorry forget to fix this one.